proto/regen/group/v1alpha1/types.proto

```diff
@@ -0,0 +1,266 @@
+syntax = "proto3";
+
+package regen.group.v1alpha1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/group";
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/any.proto";
+
+// Member represents a group member with an account address,
+// non-zero power and a comment.
+message Member {
+
+    // address is the member's account address.
+    bytes address = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
```

Comment:
Should this be `string` so as to keep consistent with how we are encoding addresses as bech32 encoded strings elsehwere in the SDK ?